userspace/falco/app/actions/print_page_size.cpp

```diff
@@ -24,7 +24,15 @@ falco::app::run_result falco::app::actions::print_page_size(falco::app::state& s
 {
 	if(s.options.print_page_size)
 	{
+#ifndef _WIN32
 		long page_size = getpagesize();
+#else
+		SYSTEM_INFO sysInfo;
+
+		GetSystemInfo(&sysInfo);
```

Comment:
Looks like there is no header file included for this function :thinking:

Reply:
I have seen this in other files as well, a bit confused as to where this is imported from

Reply:
Cc @therealbobo